src/lib/giantswarm_client_wrapper.js

```diff
@@ -3,12 +3,11 @@
 
 import { auth0Login } from 'actions/userActions';
 import { isJwtExpired } from 'lib/helpers';
+import { store } from '../components/app';
```

Comment:
importing the store that we initialise in app, instead of loading the singleton store